risc-v-demo/sev-kit-reference-design/script_support/additional_configurations/functions.tcl

```diff
@@ -37,21 +39,14 @@ proc getHlsPaths { } {
     if {![file exists "$shls_path"]} {
         puts stderr "Error: Cannot find SmartHLS (shls)."
         puts stderr "Please specify a full path to SmartHLS (shls file) using \"shls_path\" parameter in the \"script_args\"."
-        puts stderr "For example: script_args:shls_path:C:/Microchip/SmartHLS-20XX.Y.Z/SmartHLS/bin/shls"
-        exit 1
+        puts stderr "For example: script_args:shls_path:C:/Microchip/SmartHLS-2022.2.1/SmartHLS/bin/shls"
+        #exit 1
```

Comment:
Why was the exit command commented out?

Reply:
Resolved: efc36140eb5a18019730786e6ce7a9a4466c61b2